test/jdk/javax/swing/JColorChooser/Test4887836.java

```diff
@@ -21,23 +21,40 @@
  * questions.
  */
 
+import java.awt.Color;
+import java.awt.Font;
+import javax.swing.JColorChooser;
+import javax.swing.UIManager;
+
 /*
  * @test
  * @bug 4887836
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @summary Checks if no tooltip modification when no KeyStroke modifier
- * @author Konstantin Eremin
- * @run applet/manual=yesno Test4887836.html
+ * @run main/manual Test4887836
  */
+public class Test4887836 {
+
+    public static void main(String[] args) throws Exception {
+        String instructions = "If you do not see white area under the \"Swatches\" tab,\n" +
+                "then test passed, otherwise it failed.";
+
+        PassFailJFrame.builder()
+                .title("Test4759306")
+                .instructions(instructions)
+                .rows(5)
+                .columns(40)
+                .testTimeOut(10)
+                .splitUIRight(Test4887836::createColorChooser)
+                .build()
+                .awaitAndCheck();
+    }
 
-import java.awt.Color;
-import java.awt.Font;
-import javax.swing.JApplet;
-import javax.swing.JColorChooser;
-import javax.swing.UIManager;
 
-public class Test4887836 extends JApplet {
-    public void init() {
+    private static JColorChooser createColorChooser() {
+        JColorChooser chooser = new JColorChooser(Color.LIGHT_GRAY);
         UIManager.put("Label.font", new Font("Perpetua", 0, 36)); // NON-NLS: property and font names
```

Comment:
Can add comment above.
```suggestion
        // NON-NLS: property and font names
        UIManager.put("Label.font", new Font("Perpetua", 0, 36));
```